src/main/java/frc/robot/RobotContainer.java

```diff
@@ -172,6 +176,13 @@ public void updateOIType() {
         operatorOI = gamepadOI;
         driverOverrideOI = gamepadOI;
         break;
+      // case "Arduino Leonardo":
+      // System.out.println("Robot controller: Arduino");
+      // OIArduinoConsole arduinoOI = new OIArduinoConsole();
+      // // driverOI = ;
+      // operatorOI = arduinoOI;
+      // // driverOverrideOI = ;
+      // break;
```

Comment:
At some point we will need to improve this logic to detect and use the Arduino OI whenever it is available.